### DISCUSSION TITLE
Is it possible to check if pop3 message is a response to smtp message?

### DISCUSSION THREAD

**Discussion by argentario84** [jstedfast/MailKit#1671] [Q&A] [answered]
Hi all,
first of all thanks for this beautiful and useful project.

In my project i send email:
string response= client.Send(message); 
in string response i find that value (2.0.0 3a4ZrxhhIm3Dp3a4ZrbwfJ mail accepted for delivery), i check if first part is 2.0.0 to if message is correctly delivered and i save in database second part of a string.
My question is how can i see if message download with Pop3 in other part of my projects is reply for the sent message? I expected to find the previous code inside one of the fields of the pop3 message object but this doesn't seem to be the case

Thanks

**Comment by jstedfast** [comment, marked as answer]
There's no way to check for that.
